llarp/endpoint_base.hpp

```diff
@@ -1,29 +1,27 @@
 #pragma once
 
-#include "llarp/service/address.hpp"
-#include "llarp/service/convotag.hpp"
-#include "llarp/service/protocol_type.hpp"
 #include "router_id.hpp"
-#include "llarp/ev/ev.hpp"
-#include "llarp/dns/srv_data.hpp"
+
+#include <llarp/dns/srv_data.hpp>
+#include <llarp/ev/ev.hpp>
+#include <llarp/link/tunnel.hpp>
+#include <llarp/service/address.hpp>
+#include <llarp/service/convotag.hpp>
+#include <llarp/service/protocol_type.hpp>
+
+#include <oxenc/variant.h>
+#include <quic.hpp>
 
 #include <functional>
 #include <memory>
+#include <optional>
+#include <set>
 #include <string>
 #include <tuple>
-#include <optional>
 #include <unordered_set>
-#include <set>
-#include "oxenc/variant.h"
-#include <quic.hpp>
 
 namespace llarp
 {
-  namespace quic
-  {
-    class TunnelManager;
-  }
```

Comment:
This should remain as a forward declaration (include of having the `#include <llarp/link/tunnel.hpp>`), so that lokinet-core-utils doesn't have to depend on lokinet-wire (where `link/tunnel.cpp` gets compiled).